system_baseline/views/v1.py

```diff
@@ -31,6 +31,10 @@ def _get_total_available_baselines():
     """
     account_number = view_helpers.get_account_number(request)
     query = SystemBaseline.query.filter(SystemBaseline.account == account_number)
+
+    message = "reading number of baselines"
```

Comment:
Why is this one "reading number of baselines" instead of counting baselines as in other places?

Reply:
Good catch!